IANotificationsTestProject.xcodeproj/project.pbxproj

```diff
@@ -37,6 +47,19 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		BAD29F2025D08AD10054DA6F /* In App Notifications */ = {
+			isa = PBXGroup;
+			children = (
+				BAD29F1425CF88D40054DA6F /* SnackbarNoticeView.xib */,
+				BAD682BB25E491D100476BA8 /* NoticeView.swift */,
```

Comment:
Nipticky: Probably both `NoticeView.swift` and `SnackbarNoticeView.xib` should have the same name? (still reading!)

Reply:
Totally!  This is part of that problem I have been describing to you where if the names match the classes suddenly become non coding compliant when creating the properties.  I haven't been able to identify why yet, but the intention is that these will have the same name when moved into the main project